src/App.js

```diff
@@ -24,10 +25,15 @@ class App extends Component {
           </Grid>
         </Navbar>
         <Grid>
-          {/* Comment out the components to leave only the one you need to work on */}
-          <InfoPanel {...this.state} />
+          {/* Comment out the components to leave only the one you need to work on 
```

Comment:
Even though you might want to develop with this locally, we don't want these changes to be merged to master. Since you have already committed this, you should add another commit that undoes it back to what it used to be in the master branch

Reply:
In general, it's okay to change stuff like this while developing, but don't `git add` the files that shouldn't be merged to master